CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @govuk-one-login/one-login-home-dev
```

Comment:
Hi @mateen-hussain - could you remove `* @govuk-one-login/one-login-home-dev` please thanks

Reply:
Hi @MSBennett4 - if you need us to check a PR - please contact us on [#di-one-login-home](https://gds.slack.com/archives/C012GEZBZM0) and we will try to have a look